README.md

```diff
@@ -1,111 +1,175 @@
+
+When used in combination with the `org` and `user` inputs, the `addUser` option will automatically invite a specified Github user to the destination organization if they are not already a member.
+
+> 💡 **Note:** The email invitation will be sent from whichever account is used to authenticate the Github action and fork the requested repository, meaning there must be sufficient permissions to invite outside users to the organization.
+
+**Example:** `true`
+
+**Default:** `false`
+
+### `licenseWhitelist` (optional, string)
+
+A newline-delimited (`"\n"`) string representing a list of allowed [license keys](https://docs.github.com/en/rest/reference/licenses) for the repository being forked. If the license key returned by the [Licenses API](https://docs.github.com/en/rest/reference/licenses) is not found within the `licenseWhitelist`, `forker` will **not** fork the repository, and instead exit with a warning.
+
+> 💡 **Tip:** You can always reference [this directory](https://github.com/github/choosealicense.com/tree/gh-pages/_licenses) if you need a comprehensive list of license keys, beyond the commonly-used licenses returned from `GET /licenses` in the [Github REST API](https://docs.github.com/en/rest/reference/licenses#get-all-commonly-used-licenses).
+
+**Example:** `"0bsd\napache-2.0\nmit"`
+
+## Usage
+
+### Typical
+
+In most cases, you'll want to use the latest stable version (eg. `v0.0.1`):
+
+```yaml
+uses: lelia/forker@releases/v0.0.1
+with:
+  token: ${{ secrets.GH_API_TOKEN }}
+  repo: tremor-runtime
+  owner: tremor-rs
+  user: lelia
 ```
 
-Run the tests :heavy_check_mark:  
-```bash
-$ npm test
+### Development
 
- PASS  ./index.test.js
-  ✓ throws invalid number (3ms)
-  ✓ wait 500 ms (504ms)
-  ✓ test runs (95ms)
+If you're actively [developing](#Developing) a new feature for the action, you can always reference a specific commit SHA (eg. `16a9cab520b7f00e68397a7b8a4067ac40353230`):
 
-...
+```yaml
+uses: lelia/forker@899add26c0bb00f6c8366cd8c5555e9309580193
+with:
+  token: ${{ secrets.GH_API_TOKEN }}
+  repo: tremor-runtime
+  owner: tremor-rs
+  user: lelia
 ```
 
-## Change action.yml
+### Advanced
 
-The action.yml contains defines the inputs and output for your action.
+If you are automating forking on behalf of a Github organization, you may wish to leverage the optional `addUser` and `licenseWhitelist` params:
 
-Update the action.yml with your name, description, inputs and outputs for your action.
+```yaml
+uses: lelia/forker@releases/v0.0.1
+with:
+  token: ${{ secrets.GH_API_TOKEN }}
+  repo: tremor-runtime
+  owner: tremor-rs
+  org: wayfair-contribs
+  user: lelia
+  addUser: true
+  licenseWhitelist: "0bsd\napache-2.0\nmit"
+```
 
-See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-github-actions)
+## Developing
 
-## Change the Code
+> 💡 **Tip:**  You'll want to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
```

Comment:
Is 9 the cutoff? Would it make sense to list node >= 9 as a specific requirement rather than "reasonably modern"?

Reply:
No strong opinion here as this was just a note provided in the original [Actions template](https://github.com/actions/typescript-action). I can definitely update the phrasing as suggested to make it clearer.

Reply:
Updated phrasing and also included a note about typescript